inflection/src/inflection/message2/MF2Factory.hpp

```diff
@@ -13,19 +13,18 @@ class FormatterFactory;
 class SelectorFactory;
 }  // namespace U_ICU_NAMESPACE::message2
```

Comment:
Typically these internal declarations go to the forward header too.

Reply:
SelectorFactory and FormatterFactory are not internal, but external